run_robot_test.sh

```diff
@@ -16,9 +16,17 @@ JENKINS_BUILD_NUMBER=${BUILD_NUMBER:-666}
 
 # -------- functions -------------
 
+override_url_variables() {
+    # Overrides Openshift URL variables with Current Openshift URL values
+    echo "Retrieving Openshift Console & API Server URL Values"
+    console_url=$(oc whoami --show-console)
+    api_server=$(oc whoami --show-server)
+    TEST_VARIABLES="${TEST_VARIABLES} --variable OCP_CONSOLE_URL:${console_url} --variable OCP_API_URL:${api_server}"
```

Comment:
@mrrajan Will this work on DSAL machines too?

Reply:
@ikanias We could run this on the top of openshift, so it is possible.

Reply:
Thanks.